packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -98,4 +98,12 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   // Boolean cell text
   booleanCellTrueLabel: 'true',
   booleanCellFalseLabel: 'false',
+
+  // Core components keys
+  MuiTablePagination: {},
+  MuiPagination: {},
+  MuiAlert: {},
+  MuiBreadcrumbs: {},
+  MuiRating: {},
+  MuiAutocomplete: {},
```

Comment:
We don't natively use all these components. Why should we add them all?

Reply:
One question I would have is, let's say we want to implement Ant Design, as a design specification. How does these change fit it?

Reply:
We don't, correct but for some of them, it does make sense to have them.  Technically we only use the `MuiTablePagination` so we can only keep that one and add the rest on demand if we happen to use them in the future. The benefit is that that way all the localizations are kept together.

@oliviertassinari regarding the Ant Design - I didn't understand the problem - is it if we use a different pagination component?

Actually, these are only here because the way that the `getLocaleText` API works is that it throws an error if the key doesn't exist. If that was not the case we woudn't need to keep these here at all.

Reply:
I would keep only `MuiTablePagination`. If we add all of them it might create doubts for who is doing translations.

Reply:
> is it if we use a different pagination component?

Different design specs can be supported in different ways. The most likely path we will take is that we will expose the same MuiPagination structure, but styled differently. The only thing that we would need to do is to reinject the new components. So we might be able to use the same translation keys, as we would display the same content.